packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md

```diff
@@ -332,7 +340,7 @@ if (myContext.isLoginHidden) {
 
 **Solutions for the programmatic impact**
 
-Consider alternative integrations within your application. Since the IdP is known, you can redirect for IdP verification for all authentication flows, or leverage the [Web Finger API](/docs/reference/api/webfinger/). However, your integration may be limited based on context.
+Consider alternative integrations within your application. Since the IdP is known, you can redirect for IdP verification for all authentication flows, or use the [Web Finger API](/docs/reference/api/webfinger/). However, your integration may be limited based on context.
```

Comment:
```suggestion
Consider alternative integrations within your app. Since the IdP is known, you can redirect for IdP verification for all authentication flows, or use the [Web Finger API](/docs/reference/api/webfinger/). However, your integration may be limited based on context.
```